installing/disconnected_install/installing-mirroring-disconnected.adoc

```diff
@@ -16,76 +16,74 @@ include::modules/oc-mirror-about.adoc[leveloffset=+1]
+
+[id="mirroring-preparing-your-hosts_{context}"]
+== Preparing your mirror hosts
+
+In order to use the oc-mirror plugin to mirror images, you must install the plugin and create a container image registry credentials file to allow the mirroring from Red Hat to your mirror.
+
+// Installing the oc-mirror OpenShift CLI plugin
+include::modules/oc-mirror-installing-plugin.adoc[leveloffset=+2]
 
 // Configuring credentials that allow images to be mirrored
 include::modules/installation-adding-registry-pull-secret.adoc[leveloffset=+2]
 
 // Creating the image set configuration
 include::modules/oc-mirror-creating-image-set-config.adoc[leveloffset=+1]
 
+// Image set configuration parameters
+include::modules/oc-mirror-imageset-config-params.adoc[leveloffset=+2]
+
+// Image set configuration examples
+include::modules/oc-mirror-image-set-config-examples.adoc[leveloffset=+1]
+
+// Setting up for incremental mirroring
+include::modules/oc-mirror-setting-incremental-mirroring.adoc[leveloffset=+1]
+
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#oc-mirror-imageset-config-params_installing-mirroring-disconnected[Image set configuration parameters]
-* xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#oc-mirror-image-set-examples_installing-mirroring-disconnected[Image set configuration examples]
 * xref:../../updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc#update-service-overview_updating-restricted-network-cluster-osus[Using the OpenShift Update Service in a disconnected environment]
+* xref:../../cli_reference/openshift_cli/extending-cli-plugins.adoc#cli-installing-plugins_cli-extend-plugins[Installing and using CLI plugins]
 
-[id="mirroring-image-set"]
+[id="mirroring-image-set_{context}"]
 == Mirroring an image set to a mirror registry
 
-You can use the oc-mirror CLI plugin to mirror images to a mirror registry in a xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#mirroring-image-set-partial[partially disconnected environment] or in a xref:../../installing/disconnected_install/installing-mirroring-disconnected.adoc#mirroring-image-set-full[fully disconnected environment].
+To populate your mirror registry with {product-title} images, you can use one of the following scenarios:
 
-These procedures assume that you already have your mirror registry set up.
+* Partially disconnected environment:
 
-[id="mirroring-image-set-partial"]
-=== Mirroring an image set in a partially disconnected environment
+If the host can access both the internet and your mirror registry, but not your cluster nodes, you can directly mirror the content from that machine. This process is referred to as connected mirroring, or mirroring to a "partially disconnected environment".
```

Comment:
This has to rephrased accordingly. You can simply state what partially disconnected means.